src/esp/assets/RenderAssetInstanceCreationInfo.h

```diff
@@ -29,10 +29,10 @@ struct RenderAssetInstanceCreationInfo {
   RenderAssetInstanceCreationInfo() = default;
 
   RenderAssetInstanceCreationInfo(
-      const std::string& _filepath,
-      const Corrade::Containers::Optional<Magnum::Vector3>& _scale,
+      std::string _filepath,
+      Corrade::Containers::Optional<Magnum::Vector3> _scale,
```

Comment:
Here I'm afraid of another ["analysis paralysis"](https://en.wikipedia.org/wiki/Analysis_paralysis) problem, where users would need to think how big each trivially-copyable type is and then decide whether to pass by value or by `const&`, and then clang tidy having a slightly different opinion in many cases, fighting with the user.

Which is why I instead decided for consistency (at the cost of *some* perf disadvantage in certain cases on certain platforms), and with all trivially copyable types in Magnum I'm:

- passing all scalar types by value (`float`, but also `Deg`)
- passing `[Static]ArrayView`, `[Mutable]StringView` and `Reference` by value, as those are just one or two pointers
- passing `StridedArrayView` by `const&`, as it's quite big
- passing all math types by `const&`, even though e.g. `Vector2` is smaller than an `ArrayView`

This way my brain is freed from deciding this again for every function signature.